### PR TITLE
Get user information (career/expertise/personality)

### PR DESCRIPTION
**Pull request by SarahhhC**
get separate user information.

### CODE REVIEW

File: src/server/models/users.model.js [hunk 1]
```diff
@@ -66,9 +66,10 @@ let userSchema = new Schema({
   password: String,
   name: String,
   education: [educationSchema],
-  work: [workSchema],
-  job: [jobSchema],
-  help: [helpSchema],
+  experience: [experienceSchema],
+  work: [experienceSchema],
```
Comment (KyuaKwon) [inline src/server/models/users.model.js]: Why experienceSchema is duplicated?
I think work needs to be removed.

File: src/server/routes/users.route.js [hunk 2]
```diff
@@ -493,7 +568,7 @@ router.get('/mentorlist', apiProtector, user.getMentorList);
  *     }
  *
  */
-router.get('/job', apiProtector, user.getJobCategory);
+router.get('/personality', apiProtector, user.getPersonalityInfo);
 
 /**
  * @api {get} /users/getRequestStatus Request mentorMode
```
Comment (KyuaKwon) [inline src/server/routes/users.route.js]: api address seems to be modified

**Comment by KyuaKwon** [comment]
- Job -> Career
- Work -> experience
- Help -> Expertise


(Just for a note)

File: src/server/routes/users.route.js (hunk 2 above)
Comment (SarahhhC) [inline src/server/routes/users.route.js]: Right! We'll use that!

File: src/server/models/users.model.js (hunk 1 above)
Comment (KyuaKwon) [inline src/server/models/users.model.js]: Don't we get the facebook data and make another object named ```registrationData```  (user.controllers.js:245~246) ?

```
education: facebookResult.education,
experience: facebookResult.work,
```

Why the schema needs to use name ```work``` also?

Comment (SarahhhC) [inline src/server/models/users.model.js]: If we don't have schema work, it occurs problem but I don't know why...T.T

**Comment by MechanicKim** [comment]
Okay Thank you for answering my questions. LGTM

**Pull request merged**